### PR TITLE
Bug 1794108 - Graph View - Show alert ID for common alerts and highlight it in a different color

### PR DESCRIPTION
**Pull request by alexandru-io**

### CODE REVIEW

**Comment by alexandru-io** [comment]
image

**Comment by beatrice-acasandrei** [comment]
I have some suggestions:

- I would say that "(log in as a sheriff to create)" should not be displayed there since the common alert already exists. 
- Alert number for common alerts is orange and for the usual alerts is blue colored.

**Comment by alexandru-io** [comment]
> I have some suggestions:
> 
>     * I would say that "(log in as a sheriff to create)" should not be displayed there since the common alert already exists.
> 
>     * Alert number for common alerts is orange and for the usual alerts is blue colored.

isn't already orange?

**Comment by alexandru-io** [comment]
image
image

**Comment by beatrice-acasandrei** [comment]
> > ```
> > * Alert number for common alerts is orange and for the usual alerts is blue colored.
> > ```
> 
> isn't already orange?

In my opinion you should keep the original color, which is blue. You already have the "Common alert" text to differentiate the alerts and I don't see a reason to not keep the original design.

**Review by beatrice-acasandrei — CHANGES_REQUESTED** [review]
I'm still getting the error when zooming in on some graphs: "Something went wrong TypeError: status is undefined."

**Review by beatrice-acasandrei — CHANGES_REQUESTED** [review]
On my side the button "Highlight common alerts" no longer works and I can't see on the graph details for the common alerts.

File: ui/perfherder/perf-helpers/helpers.js [hunk 1]
```diff
@@ -800,9 +800,9 @@ export const createGraphData = (
         alertSummary: alertSummaries.find(
           (item) => item.push_id === dataPoint.push_id,
         ),
-        commonAlert: commonAlerts.some((item) => {
-          return item.find((alert) => alert.push_id === dataPoint.push_id);
-        }),
+        commonAlertSummary: commonAlerts[0].find(
```
Comment (esanuandra) [inline ui/perfherder/perf-helpers/helpers.js:803]: why do we search here only in first element?

Comment (alexandru-io) [reply ui/perfherder/perf-helpers/helpers.js:803]: because `commonAlertSummary` is an array with elements only on position 0

**Comment by alexandru-io** [comment]
image

**Review by esanuandra — APPROVED** [review]
looks good

**Comment by jmaher** [comment]
if you rebase this on upstream/master, you will fix the issue with circleci: builds.

**Review by beatrice-acasandrei — APPROVED** [review]

**Review by beatrice-acasandrei — APPROVED** [review]

**Review by esanuandra — APPROVED** [review]

**Comment by jmaher** [comment]
please rebase on master branch to pick up changes to fix the issue with the broken builds job.

**Pull request merged**